app/services/planning_application_search.rb

```diff
@@ -16,10 +19,11 @@ class PlanningApplicationSearch
 
   define_model_callbacks :initialize, only: :after
 
-  after_initialize :init_status
+  after_initialize :init_filter_options
 
-  def init_status
+  def init_filter_options
     self.status ||= statuses
+    self.application_type ||= application_types
```

Comment:
I've had to do the assignment of status like that as it depends on a method and at the moment it was failing using lamda

Reply:
Yeah just tried to be consistent and thought we might want to show different application types based on the user or add some condition in the future. And i'm calling `application_types` in the component rather than calling the constant like `PlanningApplicationSearch::APPLICATION_TYPES`